### PR TITLE
Migration Fixes

### PR DESCRIPTION
**Pull request by mohityadav766**
### Describe your changes:

- Cleanup Unncessary Initialization
- Added Logging for Migration steps
- Throw exception in case of Force Migration

Sample Log file 

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

[migrationfile.txt](https://github.com/open-metadata/OpenMetadata/files/15205465/migrationfile.txt)

### CODE REVIEW

File: openmetadata-service/src/main/java/org/openmetadata/service/util/AsciiTable.java [hunk 1]
```diff
@@ -12,6 +12,16 @@ public record AsciiTable(
   private static final String DEFAULT_COLUMN_NAME = "(No column name)";
   private static final String DEFAULT_NO_VALUE = "-";
 
+  private static final String logo =
```
Comment (mohityadav766) [inline openmetadata-service/src/main/java/org/openmetadata/service/util/AsciiTable.java:15]: @harshach  let me know if this needs to be removed :)

File: bootstrap/sql/migrations/native/1.2.0/mysql/schemaChanges.sql [hunk 2]
```diff
@@ -259,8 +259,8 @@ WITH CTE AS (
 DELETE FROM entity_extension_time_series WHERE temp in (SELECT temp FROM CTE WHERE RN > 1);
 ALTER TABLE entity_extension_time_series DROP COLUMN temp;
 
-ALTER TABLE entity_extension_time_series MODIFY COLUMN entityFQNHash VARCHAR(768) COLLATE ascii_bin, MODIFY COLUMN jsonSchema VARCHAR(256) COLLATE ascii_bin, MODIFY COLUMN extension VARCHAR(256) COLLATE ascii_bin,
-    ADD CONSTRAINT entity_extension_time_series_constraint UNIQUE (entityFQNHash, extension, timestamp);
+ALTER TABLE entity_extension_time_series MODIFY COLUMN entityFQNHash VARCHAR(768) COLLATE ascii_bin, MODIFY COLUMN jsonSchema VARCHAR(256) COLLATE ascii_bin, MODIFY COLUMN extension VARCHAR(256) COLLATE ascii_bin;
+ALTER TABLE entity_extension_time_series DROP CONSTRAINT entity_extension_time_series_constraint, ADD CONSTRAINT entity_extension_time_series_constraint UNIQUE (entityFQNHash, extension, timestamp);
```
Comment (harshach) [inline bootstrap/sql/migrations/native/1.2.0/mysql/schemaChanges.sql:263]: any reason we are making it into another statement?

Comment (mohityadav766) [reply bootstrap/sql/migrations/native/1.2.0/mysql/schemaChanges.sql:263]: Saw some failures for this query , removing and recreating the constraint is safe and result in a idempotent operation

File: openmetadata-service/src/main/java/org/openmetadata/service/util/AsciiTable.java (hunk 1 above)
Comment (pmbrull) [reply openmetadata-service/src/main/java/org/openmetadata/service/util/AsciiTable.java:15]: we had another one maybe we can reuse the same?

**Review by pmbrull — DISMISSED** [review]

**Review by pmbrull — APPROVED** [review]

**Pull request merged**